src/pages/SignIn.js

```diff
@@ -96,6 +96,25 @@ export default class SignIn extends Component {
             }, ['https://www.federalregister.gov/documents/2018/03/14/2018-05176/privacy-act-of-1974-system-of-records'])
           ])
         ]),
+        isAnvil() && div({ style: { lineHeight: 1.5, marginTop: '2rem', paddingTop: '1rem', borderTop: Style.standardLine } }, [
+          div({ style: { fontWeight: 500 } }, ['WARNING NOTICE']),
```

Comment:
Nit: I would rather do normal casing here, e.g., `'Warning Notice'` and apply the `textTransform: 'uppercase'` to the styles. It future proofs against the text changing and makes the idea of this div explicit in the styling.